acceptance-tests/src/test/java/tech/pegasys/ethsigner/tests/dsl/node/PantheonNode.java

```diff
@@ -158,11 +161,18 @@ private void pullPantheonImage() {
   }
 
   private String createPantheonContainer(final NodeConfiguration config) {
-    @SuppressWarnings("unstable")
-    final String genesis = Resources.getResource(config.getGenesisFilePath()).getPath();
-    LOG.info("Path to Genesis file: {}", genesis);
+    final URL resource = PantheonNode.class.getResource(config.getGenesisFilePath());
```

Comment:
splitting this whole block to get the genesis file path into seperate function would be nice